.gitignore

```diff
@@ -64,4 +64,6 @@ CMakeFiles/
 *.cmake
 !FindGMP.cmake
 Makefile
-src/extern/
+
+build/
+builddir/
```

Comment:
This is not necessary, by the way. :)

Meson, when it creates `builddir/`, also creates `builddir/./gitignore` which ignores `*`.

Reply:
hmm didn't know that, thanks :D. But at least for me it wanted to include all files of builddir

Reply:
You may be using a version of Meson older than 0.57 (the February 2021 release) on your development machine.